### PR TITLE
Order details page

### PR DESCRIPTION
**Pull request by INR4GE**

### CODE REVIEW

File: src/pages/order-item/order-item.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      orderLoading: Orders.orderLoading,
+    })
+  );
+
+  const handleTabChange = (e, newValue) => {
+    setTabValue(newValue);
+  };
+
+  const handleFormSubmit = (order)=> {
+    if(order.status!==initialValues.status&&!['CREATED','CONFIRMED'].includes(order.status)) {
```
Comment (TorskyM) [inline src/pages/order-item/order-item.js:34]: Align this condition or move it to a separate function, please

File: src/pages/order-item/order-item.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  }
+
+  return (
+    <form onSubmit={handleSubmit} className={classes.orderContainer}>
+      <Paper>
+        <Tabs value={tabValue} onChange={handleTabChange}>
+          <Tab value={0} label="Загальне"/>
+          <Tab value={1} label="Отримувач"/>
+          <Tab value={2} label="Продукти"/>
+          <Tab value={3} label="Доставка"/>
```
Comment (TorskyM) [inline src/pages/order-item/order-item.js:81]: Add these labels to the config file

File: src/pages/order-item/tabs/delivery.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    <div className={classes.delivery}>
+      <TextField
+        label='Спосіб доставки'
+        name='delivery.sentBy'
+        value={sentBy}
+        onChange={handleChange}
+        variant='outlined'
+      />
+      <div className={classes.idContainer}>
+        <label htmlFor="byCourier">Доставка кур'єром: </label>
```
Comment (TorskyM) [inline src/pages/order-item/tabs/delivery.js:21]: Please move this title to the config file

File: src/pages/order-item/tabs/delivery.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      <TextField
+        label='Номер накладної'
+        name='delivery.invoiceNumber'
+        value={invoiceNumber}
+        onChange={handleChange}
+        variant='outlined'
+      />
+      <div className={classes.dateContainer}>
+        <p>Номер відділення: {courierOffice}</p>
+        <p>Відправлено о: {moment.unix(sentOn/1000).format(' HH:mm DD/MM/YYYY ')}</p>
```
Comment (TorskyM) [inline src/pages/order-item/tabs/delivery.js:33]: Same, move these titles to the config file

File: src/pages/order-item/tabs/delivery.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+      </div>
+      <TextField
+        label='Вартість доставки'
```
Comment (TorskyM) [inline src/pages/order-item/tabs/delivery.js:36]: Same

File: src/pages/order-item/tabs/general.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+        name='cancellationReason'
+        multiline
+        variant={'outlined'}
+        label={'Причина скасування'}
+        value={cancellationReason}
+        onChange={handleChange}
+      />
+      <div className={classes.dateContainer}>
+        <p>Дата створення: {moment.unix(dateOfCreation / 1000).format(' DD/MM/YYYY z HH:mm')}</p>
+        <p>Дата оновлення: {moment.unix(lastUpdatedDate / 1000).format(' DD/MM/YYYY z HH:mm')}</p>
```
Comment (TorskyM) [inline src/pages/order-item/tabs/general.js:49]: Move this title to the config file

File: src/pages/order-item/tabs/general.js [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+      </div>
+      <TextField
+        label='Залишити коментар'
```
Comment (TorskyM) [inline src/pages/order-item/tabs/general.js:52]: Move this title to the config file

File: src/redux/orders/orders.operations.js [hunk 8]
```diff
@@ -1,7 +1,249 @@
...
+    ...on Order {
+      _id
+      status
+      user {
+        firstName
+        lastName
+        patronymicName
+        email
+        phoneNumber
+      }
```
Comment (TorskyM) [inline src/redux/orders/orders.operations.js:17]: Maybe it's better to assign this query to some variable, thus method will become more readable

File: src/pages/order-item/tabs/recipient.js [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+          label={orderRecipient[item]||''}
+          key={item}
+          variant='outlined'
+          onChange={handleChange}
+          value={user[item]||''}
+        />
+      ))}
+      <TextField
+        name='userComment'
+        label='Коментар'
```
Comment (TorskyM) [inline src/pages/order-item/tabs/recipient.js:24]: Move this title to the config file

File: src/pages/order-item/tabs/delivery-details.js [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+  const classes = useStyles()
+  return (
+    <div>
+      <Accordion TransitionProps={{ unmountOnExit: true }}>
+        <AccordionSummary
+          expandIcon={<ExpandMoreSharp />}
+          aria-controls="panel1a-content"
+          id="panel1a-header"
+        >
+          <Typography>Деталі доставки</Typography>
```
Comment (TorskyM) [inline src/pages/order-item/tabs/delivery-details.js:18]: Move this title to the config file

File: src/pages/order-item/tabs/general.js [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+  const paymentOptionsElements = paymentOptions.map(({label,value}, index)=> (
+    <MenuItem key={label} value={value} disabled={index === 0}>{label}</MenuItem>
+  ))
+
+  return (
+    <div className={classes.general}>
+      <p className={classes.idContainer}><b>ID: </b>{_id}</p>
+      <div>
+        <label htmlFor="status">Статус замовлення:</label>
```
Comment (TorskyM) [inline src/pages/order-item/tabs/general.js:24]: Same

**Review by TorskyM — APPROVED** [review]

**Review by DmytroDidukh — APPROVED** [review]
No spaces between, tabs etc. Are you used '--no-verify' during committing or you turn off linter? It's hard to read

**Review by maksym-strus — APPROVED** [review]

**Pull request merged**